docs(contributiong/code/component model): add information about internationalization (#1866)
vtex/shoreline: packages/docs/pages/guides/code/component-model.mdx

function Search(props) {
  const getMessage = useMessage()

  return <input placeholder={getMessage('search-label')} {...props}/>
}
```

### Provide overwriting capability

> In most cases, it is necessary to allow the user of the component to override the internal messages.
> 
> You do this by adding a property called `messages` and passing as argument to `useMessage`.
> The `messages` type must contain the same keys used in your files from `messages` folder.

Reviewer: What do you think?
```suggestion
Some cases require the user to override the internal messages. 

You do this by adding a property called `messages` and passing it as an argument to the `useMessage` hook.
The `messages` type must contain the same keys defined in your files from the `messages` folder.
```

Author: Done!
